dracut/parse-kickstart

```diff
@@ -41,7 +41,7 @@ from pykickstart.constants import *
 from collections import OrderedDict
 
 # Import the kickstart version.
-from pykickstart.version import DEVEL as VERSION
+from version import VERSION
```

Comment:
I guess it could be at least `kickstart_version` to avoid possible future conflicts.